### PR TITLE
[bug 1087854] Refresh Firefox button on product page

### PR DESCRIPTION
**Pull request by rehandalal**
r?

### CODE REVIEW

File: kitsune/sumo/static/js/products.js [hunk 1]
```diff
@@ -1,20 +0,0 @@
-(function($, BD) {
```
Comment (rlr) [inline kitsune/sumo/static/js/products.js]: yay! KILL KILL KILL

**Comment by rlr** [comment]
The bug says "If they already have the latest version of Firefox, we display a Refresh Firefox button along "download a fresh copy" link.". So you probably need to hide that when they aren't on the latest version of Firefox.

**Comment by rehandalal** [comment]
phew, fixed ^

File: kitsune/sumo/static/less/products.less [hunk 2]
```diff
@@ -14,6 +14,7 @@
   font-family: 'Open Sans','Droid Arabic Naskh','Mplus Fulah',sans-serif;
   position: absolute;
   right: 0;
+  text-align: center;
```
Comment (rlr) [inline kitsune/sumo/static/less/products.less]: I *think* this screwed up the download button. See http://cl.ly/image/3Z1S183d1412

left is localhost and right is prod.

**Comment by rlr** [comment]
Other than the style issue. LGTM

File: kitsune/sumo/static/less/products.less (hunk 2 above)
Comment (rehandalal) [inline kitsune/sumo/static/less/products.less]: yep that's the issue. should be fixed.

**Comment by rehandalal** [comment]
fixed ^

**Comment by rlr** [comment]
r+! :fire:

**Comment by rehandalal** [comment]
a58feaebd4c9cb5a97ef1f97fd74354894bd8dca

**Pull request closed without merging**